Modules/QuestieQuestLog.lua

```diff
@@ -0,0 +1,239 @@
+            local questLogTitle = _G['QuestLogTitle'..i]
+            questLogTitle:Width(320)
+        end
+
+        QuestLogFrame:HookScript('OnShow', function()
+            QuestLogNoQuestsText:ClearAllPoints();
+            QuestLogNoQuestsText:SetPoint("CENTER", QuestLogFrame);
+        end)
+    end
+
+    S:AddCallback('Quest', LoadSkin)
+end
+
+function QuestieQuestLog:InitializeQuestLogLevelBadges()
+    if not Questie.db.char.enableQuestLogLevelBadges then
+        return
+    end
+
+    local numEntries, numQuests = GetNumQuestLogEntries();
+
+    if (numEntries == 0) then return end
+
+    local questIndex, questLogTitle, title, level, _, isHeader, questTextFormatted, questCheck, questCheckXOfs
+    for i = 1, QUESTS_DISPLAYED, 1 do
+        questIndex = i + FauxScrollFrame_GetOffset(QuestLogListScrollFrame);
+
+        if (questIndex <= numEntries) then
+            questLogTitle = _G["QuestLogTitle"..i]
+            questCheck = _G["QuestLogTitle"..i.."Check"]
+            title, level, _, isHeader = GetQuestLogTitle(questIndex)
+
+            if (not isHeader) then
+                questTextFormatted = format("  [%d] %s", level, title)
+                questLogTitle:SetText(questTextFormatted)
+                QuestLogDummyText:SetText(questTextFormatted)
+            end
+
+            questCheck:SetPoint("LEFT", questLogTitle, "LEFT", QuestLogDummyText:GetWidth()+24, 0);
+        end
+    end
+end
+
+function QuestieQuestLog:InitializeWindow()
+    if not Questie.db.char.enableWideQuestLog then
+        return
+    end
+
+    -- Configure this as a double-wide frame to stop the UIParent trampling on it
+    UIPanelWindows["QuestLogFrame"] = { area = "override", pushable = 0, xoffset = -16, yoffset = 12, bottomClampOverride = 140+12, width = 724, height = 513, whileDead = 1 };
+
+    -- Widen the window, note that this size includes some pad on the right hand
+    -- side after the scrollbars
+    QuestLogFrame:SetWidth(724);
+    QuestLogFrame:SetHeight(513);
+
+    -- Adjust quest log title text
+    QuestLogTitleText:ClearAllPoints();
+    QuestLogTitleText:SetPoint("TOP", QuestLogFrame, "TOP", 0, -18);
+
+    -- Relocate the detail frame over to the right, and stretch it to full
+    -- height.
+    QuestLogDetailScrollFrame:ClearAllPoints();
+    QuestLogDetailScrollFrame:SetPoint("TOPLEFT", QuestLogListScrollFrame, "TOPRIGHT", 41, 0);
+    QuestLogDetailScrollFrame:SetHeight(362);
+
+    -- Relocate the 'no active quests' text
+    QuestLogNoQuestsText:ClearAllPoints();
+    QuestLogNoQuestsText:SetPoint("TOP", QuestLogListScrollFrame, 0, -90);
+
+    -- Expand the quest list to full height
+    QuestLogListScrollFrame:SetHeight(362);
+
+    -- Create the additional rows
+    local oldQuestsDisplayed = QUESTS_DISPLAYED;
+    QUESTS_DISPLAYED = QUESTS_DISPLAYED + 17;
+
+    local skinDefaultFrames = true;
+    -- Show 3 more quests when ElvUI is present
+    if (elvEnabled) then
+        local E, L, V, P, G = unpack(ElvUI)
```

Comment:
Same as above: Some proper names would be nice and also just introduce the variables needed 👍🏻